### DISCUSSION TITLE
Vaultwarden Website Link

### DISCUSSION THREAD

**Discussion by wwwdepot** [tteck/Proxmox#3104] [General]
I've noticed that the script link currently redirects to a website not affiliated with the repository team. To avoid any confusion and to maintain the integrity of the project, could you please update the link to point directly to the GitHub repository?

Script: https://helper-scripts.com/scripts?id=Vaultwarden

Link: `https://www.vaultwarden.net/`

Repo: https://github.com/dani-garcia/vaultwarden

**Comment by tteck** [comment]
Done.
